src/sentry/ingest/transaction_clusterer/tasks.py

```diff
@@ -0,0 +1,33 @@
+from typing import Any
+
+import sentry_sdk
+from django.conf import settings
+
+from sentry import features
+from sentry.tasks.base import instrumented_task
+
+from . import rules
+from .datasource import redis
+from .tree import TreeClusterer
+
+MERGE_THRESHOLD = 100
+
+
+@instrumented_task(
+    name="sentry.ingest.transaction_clusterer.tasks.run_clusterer",
+    queue="txcluster",
+    soft_timeout=60 * 60,
```

Comment:
@beezz is it OK to have a long-running celery task that iterates over (potentially) 1000s of projects, or should we spawn a separate task for each project instead?